### DISCUSSION TITLE
skynet.pcall 为什么被deprecated

### DISCUSSION THREAD

**Discussion by brucelin-cn** [cloudwu/skynet#1904] [Q&A] [answered]
我下载了skynet 1.7.0 发现
-- skynet.pcall is deprecated, use pcall directly
skynet.pcall = pcall

我的代码原来依赖 skynet.pcall 打印 错误堆栈信息，现在没有了 :)

**Comment by cloudwu** [comment, marked as answer]
你可以自己实现一个。
